modules/sdk/gradle-plugins-target-platform/README.markdown

```diff
@@ -107,34 +108,49 @@ Method | Description
 `TargetPlatformExtension onlyIf(Closure<Boolean> onlyIfClosure)` | Includes a subproject in the target platform configuration if the given closure returns `true`. The closure is evaluated at the end of the subproject configuration phase and is passed a single parameter: the subproject. If the closure returns `false`, the subproject is not included in the target platform configuration
 `TargetPlatformExtension onlyIf(Spec<Project> onlyIfSpec)` | Includes a subproject in the target platform configuration if the given spec is satisfied. The spec is evaluated at the end of the subproject configuration phase. If the spec is not satisfied, the subproject is not included in the target platform configuration.
 `TargetPlatformExtension resolveOnlyIf(Closure<Boolean> resolveOnlyIfClosure)` | Includes a subproject in the resolving process both requirements and bundles configuration if the given closure returns `true`. The closure is evaluated at the end of the subproject configuration phase and is passed a single parameter: the subproject. If the closure returns `false`, the subproject is the resolution process.
```

Comment:
@Ithildir was it intended to have parenthesis around the `both requirements and bundles configuration` statement? Maybe something like this?:
> Includes a subproject in the resolving process (including both the requirements and bundles configuration) if the given closure...

Or have I misinterpreted what we're trying to say here?

Reply:
You're right, I think we should add parentheses there!